grails-app/conf/runtime.groovy

```diff
@@ -73,6 +73,10 @@ openboxes.client.notification.autohide.delay = 8000
 // Autosave configuration (OBPIH-5493)
 openboxes.client.autosave.enabled = false
 
+// Backdata configuration (OBPIH-6332)
+openboxes.dashboard.backdate.daysOffset = 1
```

Comment:
Let's use `backdatedShipments` in these two names to match the indicators these are used for.